index.css

```diff
@@ -0,0 +1,4 @@
+header {
```

Comment:
- [ ] There is no header tag in the HTML file, Kindly style the content of the `h1` tag here.